doc/source/notebooks/advanced/variational_fourier_features.pct.py

```diff
@@ -77,14 +77,14 @@ def Kuu_matern12_fourierfeatures1d(inducing_variable, kernel, jitter=None):
         (b - a) * (tf.square(lamb) + tf.square(omegas)) / lamb / kernel.variance / two_or_four
     )  # eq. (111)
     v_cos = tf.ones_like(d_cos) / tf.sqrt(kernel.variance)  # eq. (110)
-    cosine_block = LowRank(Diag(d_cos), v_cos[:, None])
+    cosine_block = LowRank(Diag(d_cos, is_positive_definite=True), v_cos[:, None])
```

Comment:
Can you make `is_positive_definite=True` by default? If not, what is the reason for not doing it?

Reply:
`Diag` is a TensorFlow class, so we have to explicitly set this at construction time.